vite.config.js

```diff
@@ -20,12 +20,11 @@ export default ({ mode }) => {
         tStart: "%{",
         tEnd: "}%",
         replace: {
-          "COOLER_LOANS_API_ENDPOINT":
-            process.env.VITE_COOLER_LOANS_API_ENDPOINT ?
-              process.env.VITE_COOLER_LOANS_API_ENDPOINT :
-              mode === "development" ?
-                "https://olympus-cooler-loans-api-dev.web.app" : // Avoids CORS errors during local development
-                "https://olympus-cooler-loans-api-prod.web.app", // Used in production builds
+          COOLER_LOANS_API_ENDPOINT: process.env.VITE_COOLER_LOANS_API_ENDPOINT
+            ? process.env.VITE_COOLER_LOANS_API_ENDPOINT
+            : mode === "development"
+            ? "https://olympus-cooler-loans-api-dev.web.app" // Avoids CORS errors during local development
+            : "https://olympus-cooler-loans-api-prod.web.app", // Used in production builds
```

Comment:
What is hosting this? Why do we need a separate endpoint to serve Cooler Loans?

Reply:
this is for cooler loan metrics subgraph not for cooler loans. jem set this up.

Reply:
@0xJem could you comment on this piece.

Reply:
This is hosted on Google Cloud

Repo is here: https://github.com/OlympusDAO/cooler-loans-api

Calculating the timeseries data in the frontend made it very slow, so it's shoved into an API that updates the data every 5 minutes